recipes/_omnibus_toolchain.rb

```diff
@@ -19,6 +19,10 @@
 
 arch = (windows_arch_i386? || i386?) ? 'i386' : node['kernel']['machine']
 
+# node['kernel']['machine'] returns amd64 on freebsd but the toolchain
+# filename uses x86_64
+arch = "x86_64" if arch == "amd64"
```

Comment:
This seems like it should also be conditional on the platform name, as I believe ubuntu/debian still use `amd64` designation.